packages/api-types/src/language.ts

```diff
@@ -0,0 +1,43 @@
+export interface LanguageResourceIdentifier {
+  type: 'language';
+  id: string;
+}
+
+export interface Language extends LanguageResourceIdentifier {
+  attributes: {
+    name: string;
+  };
+}
+
+export interface LanguageWithLinks extends Language {
+  links: {
+    self: string;
+  };
+}
+
+export interface GetLanguagesResponseBody {
+  data: LanguageWithLinks[];
+  links: {
+    self: string;
+  };
+}
+
+export interface GetLanguageResponseBody {
+  data: LanguageWithLinks;
+}
+
+export interface PostLanguageRequestBody {
+  data: Language;
+}
+
+export interface PostLanguageResponseBody {
+  data: LanguageWithLinks;
+}
```

Comment:
The api-types package will contain all of the types for all of the HTTP routes. The api project can use these types to ensure that the routes conform to these types, and the api-client package can use these types to cast the JSON received by HTTP requests.